bokeh/server/application_context.py

```diff
@@ -15,8 +15,9 @@ class ApplicationContext(object):
         data specific to an "instance" of the application.
     '''
 
-    def __init__(self, application):
+    def __init__(self, application, io_loop=None):
```

Comment:
should io_loop have a default value? I think we'll be broken if we don't get one right